src/Files.App/Dialogs/GitHubLoginDialog.xaml

```diff
@@ -80,16 +80,11 @@
 		<Border
 			x:Name="LoginSuccessBadge"
 			Grid.Row="1"
-			Width="70"
-			Height="70"
-			Margin="8"
-			x:Load="{x:Bind ViewModel.LoginConfirmed, Mode=OneWay}"
-			Background="#6ccb5f"
-			CornerRadius="35">
-			<SymbolIcon
-				Width="52"
-				Height="52"
-				Symbol="Accept" />
+			x:Load="{x:Bind ViewModel.LoginConfirmed, Mode=OneWay}">
+			<FontIcon
+				FontSize="85"
+				Foreground="{StaticResource SystemFillColorSuccessBrush}"
```

Comment:
Should this be a ThemeResource?

Reply:
I guess that if we want to let the user fully customize Files we should do that. I'm not sure which are the best practices in this case, it might be best to ask mdta